plugin-fancymenu/lxqtfancymenu.cpp

```diff
@@ -209,6 +209,10 @@ void LXQtFancyMenu::settingsChanged()
     bool categoriesAtRight = settings()->value(QStringLiteral("categoriesAtRight"), true).toBool();
     mWindow->setCategoryPosition(categoriesAtRight ? LXQtFancyMenuCategoryPosition::Right : LXQtFancyMenuCategoryPosition::Left);
 
+    mWindow->setAutoSelection(settings()->value(QStringLiteral("autoSel"), false).toBool());
+    int delay = qBound(50, settings()->value(QStringLiteral("autoSelDelay"), 250).toInt(), 1000);
```

Comment:
250 ms is the default delay Qt uses to show a sub-menu. Since Qt's default values have been decided by a great number of devs, I think it's highly preferable to use them as the default.

Reply:
Harder than a rock you are ;)

Reply:
Just a humble protector of Qt's values ;)